doc/lua_api.txt

```diff
@@ -3440,6 +3452,15 @@ Definition tables
         light_source = 0, -- Amount of light emitted by node
         damage_per_second = 0, -- If player is inside node, this damage is caused
         node_box = {type="regular"}, -- See "Node boxes"
+        connects_to = nodenames, --[[
+        * Used for nodebox nodes with the type == "connected"
+        * Specifies to what neighboring nodes connections will be drawn
+        * e.g. `{"group:fence", "default:wood"}` or `"default:stone"` ]]
+        connect_sides = { "up", "down", "east", "f", "s" }, --[[
+        * tells connected nodebox nodes to connect only to these sides of this node,
+        * valid entries: "u" = top of node, "d" = bottom, "n" | "b" = back side,
+        * "s" | "f" = front side, "e" | "l" = left side, "w" | "r" = right side.
+        * additional characters are ignored. ]]
```

Comment:
Why is only the first character checked?  This is the only API with these semantics AFAIK and it causes collisions (eg, `u = top, d = bottom`, it would make much more sense to use `t`/`b` but `b` is used for `back`).
I think other entries should at least print a warning, otherwise it's easy to miss bugs (eg, misspelling one of the names).